synapse/lib/ast.py

```diff
@@ -235,16 +235,22 @@ async def lookgenr():
             if not tokns:
                 return
 
+            buidset = set()
```

Comment:
Should we be using a spooled set?

Reply:
When we discussed this a few days ago the decision was no since the lookup input should be relatively bounded.  Also noticed that uniq uses an in-memory set.